cmd/goal/asset.go

```diff
@@ -189,6 +202,46 @@ var createAssetCmd = &cobra.Command{
 		client := ensureFullClient(dataDir)
 		accountList := makeAccountsList(dataDir)
 		creator := accountList.getAddressByName(assetCreator)
+		manager := creator
+		reserve := creator
+		freezer := creator
+		clawback := creator
+
+		if cmd.Flags().Changed("manager") {
+			assetManager = accountList.getAddressByName(assetManager)
+			manager = assetManager
+		}
+
+		if cmd.Flags().Changed("no-manager") {
```

Comment:
I suppose it would be nice to have a check to see if both switches, "manager " and "no-manager", are used, and if so, error out.  (Same for others too.)

Reply:
Done. For the flag checks, I used `cmd.Flags().Changed("manager") && cmd.Flags().Changed("no-manager")` instead of `cmd.Flags().Changed("manager") && assetNoManager` because it looks nicer with two flags on the same line.... 😬

Reply:
You can also use the variable for the manager flag directly, so you could do something as simple as `assetManager != "" && assetNoManager` without any use of `cmd.Flags().Changed` because they both have defaults set, so you would report the error if they both differ from their default.

I prefer to use the variables directly and only use `Changed` if I need to distinguish between when the switch gets its default value because it was not given on the command line, vs being given on the command-line, but being the same as the default.  Here, the distinction does not matter.